crates/polars-time/src/chunkedarray/string/infer.rs

```diff
@@ -21,12 +21,12 @@ const DATETIME_DMY_PATTERN: &str = r#"(?x)
         (?:\d{4,})                   # year
         (?:
             [T\ ]                    # separator
-            (?:\d{2})                # hour
+            (?:\d{1,2})                # hour
```

Comment:
FYI: `\d{1,2}` can be (slightly) better written as `\d{,2}` :)

Reply:
> FYI: `\d{1,2}` can be (slightly) better written as `\d{,2}` ;)

That doesn't seem to be a correct syntax for regex ?

Reply:
Argh - I was thinking of `\d{2,}` - which would be wrong anyway. Ignore me 🤣